### DISCUSSION TITLE
validation not respecting step attribute in FluentNumberField

### DISCUSSION THREAD

**Discussion by eriksena16** [microsoft/fluentui-blazor#1310] [Q&A]
I'm trying to insert decimal numbers, but when submitting, <DataAnnotationsValidator /> sends a message warning that it only accepts integer values.
![FluentNumberFieldDecimal](https://github.com/microsoft/fluentui-blazor/assets/26628113/4cb6b125-ef65-414c-b800-bc13f27fef82)

**Comment by vnbaaij** [comment]
Search through earlier (closed) issues and discussios. You will find pointers there.
